src/jump_moi_overloads.jl

```diff
@@ -14,11 +14,31 @@ function MOI.get(model::JuMP.Model, attr::BackwardOutObjective)
 end
 
 function MOI.get(model::JuMP.Model, attr::BackwardOutConstraint, con_ref::JuMP.ConstraintRef)
-    check_belongs_to_model(con_ref, model)
+    JuMP.check_belongs_to_model(con_ref, model)
     moi_func = MOI.get(JuMP.backend(model), attr, JuMP.index(con_ref))
     return JuMP.jump_function(model, moi_func)
 end
 
+# FIXME Workaround for https://github.com/jump-dev/JuMP.jl/issues/2797
+function _moi_get_result(model::MOI.ModelLike, args...)
```

Comment:
Why'd you add `_moi_get_result`?

Reply:
I wanted to call it from line 39 but I forgot. It seems best to do it to keep a consistent behavior. A solution is still needed so the status shouldn't be `OPTIMIZE_NOT_CALLED`.